src/library/library.h

```diff
@@ -95,6 +95,7 @@ class Library : public QObject {
   signals:
     void showTrackModel(QAbstractItemModel* model);
     void switchToView(const QString& view);
+    void updateTrackModel();
```

Comment:
we should do this everytime tracks are removed from trackDAO. Afaik only then there is a chance that we have stale data in a trackModel.  Would `forceTrackModelUpdate` be a better name for you?